kuma/wiki/tests/test_content.py

```diff
@@ -994,6 +994,45 @@ def test_annotate_links_external_link():
     assert normalize_html(actual_raw) == expected
 
 
+@pytest.mark.parametrize(
+    "code",
+    (
+        # Simple existing class value
+        [
+            '<pre class="syntaxbox">h1 { color:brown }</pre>',
+            '<pre class="syntaxbox notranslate">h1 { color:brown }</pre>',
+        ],
+        # No class attribute at all
+        [
+            "<pre>h1 { color:pink }</pre>",
+            '<pre class="notranslate">h1 { color:pink }</pre>',
+        ],
+        # 'notranslate' already set
+        [
+            '<pre class="notranslate brush:js">h1 { color:brown }</pre>',
+            # Note, no difference!
+            '<pre class="notranslate brush:js">h1 { color:brown }</pre>',
+        ],
+        [
+            '<pre class="NOTranslate">h1 { color:brown }</pre>',
+            '<pre class="NOTranslate">h1 { color:brown }</pre>',
```

Comment:
It looks like you forgot to add the `notranslate` in the expected result here:
```suggestion
            '<pre class="NOTranslate notranslate">h1 { color:brown }</pre>',
```

And, by the way, thanks for adding a test for that case!